src/components/sidebar/lineView/LineHeaderTable.tsx

```diff
@@ -60,42 +72,41 @@ class LineTopicTable extends React.Component<ILineTopicListProps> {
                                 </th>
                                 <th className={s.columnHeader} />
                             </tr>
-                            {this.props.lineTopics.map(
-                                (lineTopic: ILineTopic, index: number) => {
-                                    const isCurrentLineTopic = _.isEqual(
-                                        this.props.currentLineTopic,
-                                        lineTopic
+                            {this.props.lineHeaders.map(
```

Comment:
Maybe move this to own function to make render-function easier to read. Looks a bit messy right now but the reason for this might also be the current row length limit.

{ this.lineHeaderRows() }